src/install/redis/host/atlas.mdx

```diff
@@ -0,0 +1,13 @@
+---
+componentType: default
+headingText: Prepare Atlas 
+---
+
+
+Using the Atlas CLI run the following command to create a new user, and assign `clusterMonitor` and `readAnyDatabase` roles to the user (adjust the `username` and `password` as required):
+
+```
+atlas dbuser create -u username -p password --role clusterMonitor,readAnyDatabase
+```
+
+Alternatively, you can use the Redis Atlas UI to create your user and assign the same specific privileges. For more, information check the [Redis Atlas documentation](https://www.Redis.com/docs/atlas/security-add-Redis-users/#add-database-users).
```

Comment:
I wonder if we should add this to the top in a more narrative way. I'm imagining someone who would've preferred this method (for whatever reason) but didn't realize that this was an option until after running the CLI. 

Maybe you can preface the step by saying users need to create users and assign roles to accomplish X. They can do this using the Atlas CLI tool, or by navigating away from this install doc to work in the Redis Atlas UI.

I'm sure this is outside of scope, but are there other steps they can do from the UI? Maybe if we send them away, we should either specify that they can accomplish the integration thru the Redis UI, or maybe let them know why they need to return to the doc (to finish doing X thing to accomplish their Y goal)